ReactWindows/ReactNative.Net46/ReactNative.Net46.csproj

```diff
@@ -166,6 +166,7 @@
     <Compile Include="Views\Progress\ReactProgressBarViewManager.cs" />
     <Compile Include="Views\Scroll\ReactScrollViewManager.cs" />
     <Compile Include="Views\Slider\ReactSliderManager.cs" />
+    <Compile Include="Views\TextInput\PlaceholderAdorner.cs" />
```

Comment:
Adding a new file that contains code that creates an adorner that is used to draw the placeholder text...